_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -958,7 +1107,19 @@ link:https://docs.aws.amazon.com/general/latest/gr/aws_tasks-that-require-root.h
 the use of root].  Fortunately, most of these activities are rare, so usage of the root account can be kept to
 a minimum.
 
-Next, continue with the <<configure_logging>> section.
+[[configure_storage]]
+=== Configure storage
+
+[[s3_buckets_deployment]]
+==== S3 Buckets
+To make sure your S3 buckets are compliant with the benchmark, all you have to do is use the
+link:https://github.com/gruntwork-io/terraform-aws-security/tree/master/modules/private-s3-bucket[`private-s3-bucket` module].
+
+[[encrypt_ebs_volumes]]
+==== Encrypt EBS volumes
+
+Use the link:https://github.com/gruntwork-io/terraform-aws-security/tree/master/modules/ebs-encryption-multi-region[`ebs-encryption-multi-region` module]
+to configure AWS EBS encryption in all enabled regions of an AWS Account, as per the Benchmark recommendation.
```

Comment:
Same: add a `terragrunt.hcl` example?

Reply:
Added the example [here: ](https://github.com/gruntwork-io/gruntwork-io.github.io/pull/392/commits/b1041b233908639b6ac7720ae690184e2e983520)

Reply:
Just a heads up - I've reworked this for Terragrunt. I'll resolve.